draft-ietf-mls-architecture.md

```diff
@@ -1641,7 +1649,8 @@ read and write messages protected by the protocol for that Group.
 
 Details about how to verify the identity of a client depend on the MLS
 Credential type used. For example, cryptographic verification of credentials can
-be largely performed autonomously on the clients for the `x509` Credential
+be largely performed autonomously (e.g. without additional network traffic) by
```

Comment:
```suggestion
be largely performed automatically by
```
Trying to convey that X.509 is not manual, not that it doesn't require network (even though it may not)

Reply:
I was trying to convey that it doesn't require any additional network traffic, as this was the genesis of the original comment/question.

Reply:
X.509 verification does require network traffic though, to check revocation. The "In contrast" below is talking about out-of-band / manual verification